src/App.js

```diff
@@ -1,19 +1,30 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import UploadPageContainer from './components/UploadPageContainer';
+import { Provider } from 'react-redux';
+import { createStore, applyMiddleware } from 'redux';
+import loadedState from './initialState';
+import thunk from 'redux-thunk';
+import rootReducer from './reducers';
+
+window.sessionStorage.clear();
```

Comment:
This can be removed now

Reply:
The sessionStorage.clear is clearing out any session storage when you come to the page...that way you don't accidentally have leftover session information from a previous session.

Reply:
Right but let's just plan on moving it to being the last step of the upload progress. The session is over and you could even fire window.location.reload() along with it